kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_lagrange_multiplier.h

```diff
@@ -379,24 +377,20 @@ class ResidualBasedBlockBuilderAndSolverWithLagrangeMultiplier
         const auto it_dof_begin = BaseType::mDofSet.begin();
 
         //NOTE: dofs are assumed to be numbered consecutively in the BlockBuilderAndSolver
-        #pragma omp parallel for
-        for (int k = 0; k<ndofs; k++) {
-            auto it_dof =  it_dof_begin + k;
-
+        IndexPartition<std::size_t>(ndofs).for_each([&](std::size_t Index){
```

Comment:
wouldn't it be better to use the block version here?

Reply:
I agree

Reply:
Sure, I thought this before, but didn't know about the container available for DOFs ..now I found it. 
I am using block_for_each() with container `Dof<double>`. 
Will make those changes now. 
Thanks @RiccardoRossi & @philbucher